Microsoft.Toolkit.Uwp.UI/Helpers/ThemeListener.cs

```diff
@@ -58,6 +65,8 @@ public ThemeListener()
             CurrentTheme = Application.Current.RequestedTheme;
             IsHighContrast = _accessible.HighContrast;
 
+            _dispatcherQueue = DispatcherQueue.GetForCurrentThread();
```

Comment:
>GetForCurrentThread [](start = 47, length = 19)

Make sure to update docs and samples to explain that ThemeListener must be created on a UI thread.

Reply:
I've changed them so they can be set after creation as well. We'll call it out on the docs.